ruby-challenges.rb

```diff
@@ -4,31 +4,62 @@
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
 num1 = 7
-# Expected output: '7 is odd'
 num2 = 42
-# Expected output: '42 is even'
 num3 = 221
-# Expected output: '221 is odd'
 
 
+def even_or_odd num
+    if num % 2 == 0
```

Comment:
Great work on your logic!  Ruby also has a built-in method called .even? that does some of this work for you as well.